website/docs/cli/commands/auth/tutorials/configuring-geodesic.mdx

```diff
@@ -0,0 +1,255 @@
+
+- Geodesic toolbox set up for your infrastructure
+- `atmos.yaml` configured with auth providers and identities
+- Understanding of Geodesic's Dockerfile and Makefile structure
+- Atmos installed on your **host machine** (laptop/workstation) for authentication
+
+:::note Keychain Integration
+Atmos can optionally store refresh tokens in your operating system's keyring (macOS Keychain, Linux Secret Service, Windows Credential Manager) using `atmos auth user configure`. However, this keyring integration is **only available on the host machine**. Inside the Geodesic container, Atmos will use file-based credential storage in `~/.aws/atmos/`. See [Credential Storage](/cli/commands/auth/usage#credential-storage) for details.
+:::
+
+## Configuration Steps
+
+### 1. Update Your Geodesic Dockerfile
+
+Add the following environment variables to your Geodesic Dockerfile to configure AWS authentication settings:
+
+```dockerfile
+# Atmos auth configuration
+# These values must match your atmos.yaml auth configuration
+ARG ATMOS_AUTH_PROVIDER="acme-sso"
+ARG ATMOS_AUTH_IDENTITY="acme-identity"
+
+ENV AWS_SHARED_CREDENTIALS_FILE=$HOME/.aws/atmos/${ATMOS_AUTH_PROVIDER}/credentials
+ENV AWS_CONFIG_FILE=$HOME/.aws/atmos/${ATMOS_AUTH_PROVIDER}/config
+ENV AWS_PROFILE=${ATMOS_AUTH_IDENTITY}
+ENV ATMOS_IDENTITY=${ATMOS_AUTH_IDENTITY}
+
+# Path to the teams config file for assume-role and other Geodesic utilities.
+# Once logged in with Atmos, this config file gives you access to all the other teams
+# and roles (if you are authorized for access).
+ENV AWS_CONFIG_TEAMS=/etc/aws-config/aws-config-teams
+ENV ASSUME_ROLE_INTERACTIVE_QUERY=${NAMESPACE}${TENANT:+-$TENANT}-gbl-
+```
+
+**Key Configuration Details:**
+
+- **`ATMOS_AUTH_PROVIDER`**: Name of the provider defined in your `atmos.yaml` (e.g., `acme-sso`) - used to construct credential file paths
+- **`ATMOS_AUTH_IDENTITY`**: Name of the identity to use by default (must match an identity in `atmos.yaml`)
+- **`AWS_SHARED_CREDENTIALS_FILE`**: Points to Atmos-managed credentials directory
+- **`AWS_CONFIG_FILE`**: Points to Atmos-managed AWS config file
+- **`AWS_PROFILE`**: The AWS profile name (set to match identity)
+- **`ATMOS_IDENTITY`**: Environment variable that Atmos reads to determine default identity
+- **`AWS_CONFIG_TEAMS`**: Separate variable for teams config (avoids clobbering `AWS_CONFIG_FILE`)
+
+:::note
+The `AWS_CONFIG_FILE` environment variable must remain pointed at the Atmos-managed config (`$HOME/.aws/atmos/${ATMOS_AUTH_PROVIDER}/config`) for authentication to work correctly. The teams config is stored in the separate `AWS_CONFIG_TEAMS` variable. If your Geodesic utilities reference the teams config, update them to use `AWS_CONFIG_TEAMS` instead of `AWS_CONFIG_FILE`.
+:::
+
+### 2. Update Your Makefile
+
+Add an automatic login step to your Makefile to ensure users are authenticated before starting the Geodesic shell:
+
+```makefile
+login:
+	@atmos auth login --identity acme-identity
```

Comment:
This won't work because we require the browser to complete the authentication. 

I would remove this section and instead ask the user to run atmos before make

Reply:
resolved with https://github.com/cloudposse/atmos/pull/1671